packages/vx-grid/src/grids/GridRows.tsx

```diff
@@ -3,22 +3,25 @@ import cx from 'classnames';
 import Line, { LineProps } from '@vx/shape/lib/shapes/Line';
 import { Group } from '@vx/group';
 import { Point } from '@vx/point';
-import { Scale, CommonGridProps } from '../types';
+import { getTicks, ScaleInput } from '@vx/scale';
+import { CommonGridProps, GridScale } from '../types';
 
-export type GridRowsProps<ScaleInput> = CommonGridProps & {
+export type GridRowsProps<Scale extends GridScale> = CommonGridProps & {
   /** `@vx/scale` or `d3-scale` object used to map from ScaleInput to y-coordinates. */
-  scale: Scale<ScaleInput, number>;
+  scale: Scale;
+  /** Exact values used to generate grid coordinates (y- for Rows, x- for Columns) lines using `scale`. Overrides `numTicks` if specified. */
```

Comment:
should we remove the `(y- for Rows, x- for Columns)` piece here? also I think a leftover typo from me, could likely also remove the word `coordinates` here